### ISSUE TITLE
model without ivector problem

### ISSUE THREAD

**Issue by uouo111845** [alphacep/kaldi-android-demo#28]
Hi
I want to know whether this kaldi-android-demo support a model which is not trained with ivector feature. My model works well in this demo, but its size only for the ivector directory is 20M, so I want to test a new model without ivector.

**Comment by nshmyrev** [comment]
Yes, it should work without ivector, just don't configure it in the model.cc, that will be ok. This is a good idea actually, I'm also experimenting on decoding without ivectors due to ivector stability problems. For now ivector is still required though. In our model I train extractor of dim 30 instead of standard 100, that makes extractor smaller and more stable.
